src/NUnitFramework/testdata/LifeCycleFixture.cs

```diff
@@ -52,26 +49,41 @@ public void Dispose()
     [FixtureLifeCycle(LifeCycle.InstancePerTestCase)]
     public class SetupAndTearDownFixtureInstancePerTestCase
     {
-        public int TotalSetupCount = 0;
-        public int TotalTearDownCount = 0;
+        int _totalSetupCount = 0;
```

Comment:
I did this because the test was wrong, it was creating an instance of this class then running it directly. That isn't quite correct because we should have an instance created for each test method, so we can't check these variables because they only apply to the initial instance. Instead, I rewrote the test to check itself then check the test result.